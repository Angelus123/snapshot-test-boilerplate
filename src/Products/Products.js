import React from 'react';

// #region constants

// #endregion

// #region styled-components

// #endregion

// #region functions

// #endregion

// #region component
const propTypes = {};

const defaultProps = {};

/**
 * 
 */
const Products = ({ items = [] }) => {
    if (items.length <= 0) {
        return <div></div>;

    } else {
        return <div>
            {items.map((item, index) => {
                return <h4>{item} </h4>
            })}
        </div>
    }
    
}

Products.propTypes = propTypes;
Products.defaultProps = defaultProps;
// #endregion

export default Products;