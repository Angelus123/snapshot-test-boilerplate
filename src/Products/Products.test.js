import React from 'react';
import renderer from 'react-test-renderer';
import Products from './Products';

it ("render when there are no items", () => {
    const tree =renderer.create(<Products />).toJSON()
    expect(tree).toMatchSnapshot();
})