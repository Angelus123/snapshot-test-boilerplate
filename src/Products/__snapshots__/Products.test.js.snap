// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render when there are no items 1`] = `<div />`;
